_posts/2023-04-04-flux-accounting-0.23.0.md

```diff
@@ -0,0 +1,21 @@
+---
+title: flux-accounting v0.23.0
+date: 2023-04-04 00:00:00
+author: "flux-framework"
+categories: 'release'
+version: 0.23.0
+download: https://github.com/flux-framework/flux-accounting/releases/download/v0.23.0/flux-accounting-0.23.0.tar.gz
+---
+
+Download from GitHub [here]({{ page.download }})
+
+# Release Notes
+
+flux-accounting version 0.23.0 - 2023-04-04
+-------------------------------------------
+
+#### Fixes
+
+* bindings: raise error to caller ([#327](https://github.com/flux-framework/flux-accounting/issues/327), [#328](https://github.com/flux-framework/flux-accounting/issues/328), [#329](https://github.com/flux-framework/flux-accounting/issues/329))
```

Comment:
@grondo @cmoussa1 how does this look? I didn't do this with a regular expression, I did basic parsing based on finding the numbers in two parens. So - for future notes please try to confirm to that general structure!

And since this is new, I will watch it closely for any bugs / issues.

Reply:
This LGTM! Including numbers in between two parens is how I reference all PRs, so you can expect me to continue to follow that structure! If not, you have permission to yell at me. 😅 

I should let @grondo give his thoughts too, since I typically format the release after flux-core's style.